1주차/4673.py

```diff
@@ -0,0 +1,21 @@
+#4673
+
+num_list=[]
+for i in range(1,10001):
+  num_str=str(i)
+  for j in range(len(num_str)):
+    i+= int(num_str[j])
```

Comment:
```suggestion
  for j in num_str:
    i+= int(j)
```
이렇게 좀 더 파이썬스러운 for문으로 작성해보세요.

string, list, tuple 같은것들을 for문에서 각 원소를 하나씩 차례대로 꺼내서 쓰고싶을땐

```
for i in range(len(arr)):
    print(arr[i])
```
보다는
```
for i in arr:
    print(i)
```
이런 식으로 for문을 써보세요